marklogic-client-api/src/main/java/com/marklogic/client/expression/PlanBuilder.java

```diff
@@ -599,11 +599,47 @@ protected PlanBuilder(
   * @return  a ModifyPlan object
   */
   public abstract ModifyPlan fromSql(XsStringVal select, XsStringVal qualifierName);
+  /**
+  * Starts a pipeline with rows provided by a parameter when the pipeline is executed.
+  * @param paramName  Specifies the placeholder parameter supplying the rows.
+  * @param qualifier  Specifies a view name alias similar to op.fromLiterals() and other accessors.
```

Comment:
I think we should reuse the description found elsewhere for `qualifier` - i.e. "Specifies a name for qualifying the column names similar to a view name". Stating "view name alias" is a little confusing to me because it suggests that a view is required.